### DISCUSSION TITLE
How to update session in Auth.js v5?

### DISCUSSION THREAD

**Discussion by patrickariel** [nextauthjs/next-auth#11217] [Help]
There has been a few issues documenting this problem, but they were all closed:

- https://github.com/nextauthjs/next-auth/issues/11002
- https://github.com/nextauthjs/next-auth/issues/11018

So I'm wondering, is `update` working for anyone at all? My use-case was that I was trying to update the name field in the database and updating the session to reflect it. I already have a working database adapter setup, but the `name` field only gets filled after logging out and then logging back in.

Anyone have any solutions?

**Comment by asayushranjansinha** [comment]
Did you find out the solution to the problem for your use case? I am having a similar problem and can't seem to figure out.

**Comment by maxpaleo** [comment]
This is such a mess, as with everything in Auth.js... session.update() indeed doesn't work... The latest docs don't mention updating the session at all whatsoever, the only reference is the [UpdateSession()](https://authjs.dev/reference/nextjs/react#updatesession) , but it doesn't actually update anything and the "trigger" event in the jwt or session callbacks is always null...
